### PR TITLE
feat: Noosphere docs website skeleton

### PR DESCRIPTION
**Pull request by cdata**
This change proposes the introduction of an [11ty](https://11ty.dev)-based TypeScript package that produces a static website intended to serve as an introduction and reference manual for the Noosphere Protocol.

The site that is currently generated is a skeleton, and lacks most of the nominal reference material. But, we can still check it in and get it publishing on Github, and that will give us a common baseline to iterate from as we seek to fill out our docs.

Demo (with no guarantee of future availability after merge): https://cdata.github.io/noosphere-guide-demo/

### CODE REVIEW

File: typescript/packages/noosphere-guide/get-started.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+# Get started with Noosphere
+
+Noosphere supports a few different programming environments, including:
+
+- [Rust][noosphere-rust-docs]
+- <a target="_blank">TypeScript (web, via NPM)</a>
+- <a target="_blank">Swift</a>
+- <a target="_blank">C</a>
```
Comment (justinabrahms) [inline typescript/packages/noosphere-guide/get-started.md:12]:
```suggestion
- TypeScript (web, via NPM)
- Swift
- C
```

don't make it look like a link if it's not a link? We can add it later.

File: typescript/packages/noosphere-guide/docs.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+- [End-to-end encryption](https://en.wikipedia.org/wiki/End-to-end_encryption)
+
+Above this substructure, Noosphere gives users:
+
+- Entry to a zero-trust, **decentralized network** of self-sovereign nodes
+- **Human-readable names** for peers and their public content
+- **Local-first authoring** and offline-available content with conflict-free synchronization
+- A complete, space-efficient **revision history** for any content
+- Coherence and **compatibility with the hypertext web**
+ -->
```
Comment (justinabrahms) [inline typescript/packages/noosphere-guide/docs.md:73]:
```suggestion
```

**Review by justinabrahms — APPROVED** [review]
Minor comments, but seems fine to me. Would love to have this show a preview of what it looks like (similar to how netlify does)

**Comment by cdata** [comment]
> Would love to have this show a preview of what it looks like (similar to how netlify does)

This sounds awesome. Is it something we can layer on top, perhaps?

**Comment by justinabrahms** [comment]
For sure.

**Comment by cdata** [comment]
@justinabrahms I made a one-off demo on a personal repo, which was good because I caught a few issues related to relative paths and Github's built-in default Jekyll behavior. Demo is here: https://cdata.github.io/noosphere-guide-demo/

File: typescript/packages/noosphere-guide/get-started.md (hunk 1 above)
Comment (cdata) [reply typescript/packages/noosphere-guide/get-started.md:12]: Check the preview. I have a CSS-based thing for this.

**Comment by justinabrahms** [comment]
Header is a little wonky?

**Comment by cdata** [comment]
Hrm, looks like Firefox doesn't like the linear gradient BG......

**Comment by cdata** [comment]
Thanks for catching that @justinabrahms it should be fixed now.

**Review by justinabrahms — APPROVED** [review]

**Pull request merged**